packages/server/lib/server-base.ts

```diff
@@ -117,13 +134,13 @@ export abstract class ServerBase<TSocket extends SocketE2E | SocketCt> {
   protected _netStubbingState?: NetStubbingState
   protected _httpsProxy?: httpsProxy
 
-  protected _remoteAuth: unknown
```

Comment:
I dunno about these types, it's not clear at all what they are, slowing figuring it out.